scss-partials/_hero.scss

```diff
@@ -0,0 +1,56 @@
+.hero {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    position: relative;
+
+    &__img {
+        width: 100%;
+        height: auto;
+
+        @include media ( "<md" ) {
+            height: 432px;
+            object-fit: cover;
+            object-position: 45%;
+        }
+    }
+
+    &__title {
+        text-align: center;
+        font-family: $font-heading;
+        font-size: 32px;
+        margin: 0 auto;
+        position: absolute;
+        font-weight: 300;
+        width: 100%;
+
+        @include media ( ">=md" ) {
+            font-size: 48px;
+        }
+    }
+
+    &__title--withShadow {
```

Comment:
here's a perfect case of a modifier :)  👍  you could add it here like this
```
  &---withShadow 
```
directly below the &__title

Reply:
Have updated now, does this look ok?